compiler/luci/lang/include/luci/IR/Nodes/CircleLQFullyConnected.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
It's needed to update year here?

Reply:
Yes, this is new file, so you need to add year of initial commit here.